__snapshots__/src/index.test.js.md

```diff
@@ -177,12 +180,15 @@ Generated by [AVA](https://ava.li).
         className="dropdown"
       >
         <Trigger
+          clientId="rdts"
           disabled={true}
           onTrigger={Function {}}
           showDropdown={false}
+          tags={[]}
```

Comment:
Shouldn’t there be tags here instead of an empty array?

Reply:
Not sure I follow. It’s the same state.tags as sent to the input, which is empty?

Reply:
Seems so.

Reply:
https://github.com/dowjones/react-dropdown-tree-select/blob/8eefed776959458a657c90b48d0360252fb1eb63/src/index.test.js#L28
Do you want me to set a node as checked/selected?

Reply:
I was hoping to look at this last weekend but didn’t get a chance. I think I’m not understanding the intent here. 

What I assumed previously is that this change is supposed to populate tags as aria attributes in the rendered markup. The snapshot shows empty so I thought that was a mistake. From your last comment, it seems that happens only when you check a node. If so, then I think we there should be a test to verify that behavior (which I think you already have added). If all of that is true, then there is nothing to do here and we can mark this as resolved.

Reply:
The snapshot was changed because the input needs info about which nodes are checked/tagged. No intent behind this specific tests. The output (aria-attributed) is verified in other tests.